core/src/main/scala/org/apache/spark/rapids/tool/benchmarks/SingleThreadedProfileToolBenchmark.scala

```diff
@@ -0,0 +1,35 @@
+/*
+ * Copyright (c) 2024, NVIDIA CORPORATION.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package org.apache.spark.rapids.tool.benchmarks
+
+import com.nvidia.spark.rapids.tool.profiling.{ProfileArgs, ProfileMain}
+
+object ProfToolBenchmark extends BenchmarkBase {
+  override def runBenchmarkSuite(inputArgs: Array[String]): Unit = {
+    // Currently the input arguments are assumed to be common across cases
+    // This will be improved in a follow up PR to enable passing as a config
+    // file with argument support for different cases
+    runBenchmark("Benchmark_Per_SQL_Arg_Profiling") {
```

Comment:
There is no "PER_SQL" argument for Profiling. that prefix was used in the qualification's benchmark because we were running the benchmark with the perSql argument enabled/disabled.
Suggestion is:

- to enable CSV and call it something like Benchmark_Profiling_CSV
- if this is supposed to run a single thread, then the number of threads should be specified. Otherwise, the benchmark will have non-deterministic behavior for multiple eventlogs.